src/nnfusion/engine/pass/graph/kernel_tuning.cpp

```diff
@@ -21,28 +21,12 @@ DEFINE_string(ftuning_blocklist,
               "",
               "List of op types that skip kernel tuning pass, e.g., \"Softmax,Add\"");
 DEFINE_string(fantares_perf_file, "./antares_perf.csv", "File to save Antares kernel performance.");
+DEFINE_string(ftuning_platform, "", "Antares platform: e.g., win64, xbox, etc.");
```

Comment:
It's easy to make mistake with reasonable but nonexistent combination, like "device_type=cuda + platfom=linux"

Reply:
yes, this strong related to antares backend naming: https://github.com/microsoft/antares/tree/v0.3.x/backends　
maybe we can add this list or link to our document to avoid confusion.